content/logs/_index.md

```diff
@@ -209,7 +211,7 @@ logs:
 
 ### Search and replace content in your logs
 
-If your logs contain sensitive information that you wish you redact, you can configure sequences to mask in your configuration file. This is accomplished by using the `log_processing_rules` parameter in your configuration file with the **mask_sequences** `type`.
+If your logs contain sensitive information that you wish you redact, configure the Datadog Agent to mask sensitive sequences by using the `log_processing_rules` parameter in your configuration file with the **mask_sequences** `type`.
```

Comment:
we call it Datadog Agent here but Log Agent on line 155

Reply:
👍